### PR TITLE
[plugin/test] Plugin test with jdk14

### PR DESCRIPTION
**Pull request by dmsolr**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bug fix
- [x] New feature provided
- [ ] Improve performance

- Related issues

#5338 

1. Provided a test case to verify the `record`(a new feature of JDK14)
2. Introduce a new docker image(agent-jvm-container:jdk14-version) for support JDK14.
3. Remove the meaningless environment variable, `BUILD_NO`, since we move the CI to Github Action.
4. Now, the `agent-jvm-container/agent-tomcat-container` image version is the same as module(test/plugin/containers) version.
5. Specify `image_version` as `jdk14-1.0.0` to use JDK14 image. For instance: 
```shell
bash test/plugin/run.sh --image_version=jdk14-1.0.0 jdk14-with-gson-scenario
```

### CODE REVIEW

File: test/plugin/scenarios/jdk14-with-gson-scenario/support-version.list [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# lists your version here
+
+2.8.6
```
Comment (wu-sheng) [inline test/plugin/scenarios/jdk14-with-gson-scenario/support-version.list:19]: One version should be enough, we don't need to test GSON here :)

File: test/plugin/containers/jvm-container/pom.xml [hunk 2]
```diff
@@ -32,8 +32,63 @@
...
+                                <runCmds>
+                                    <run>chmod +x /usr/local/skywalking/run.sh</run>
+                                    <run>tar -xvf ../tools/skywalking-mock-collector.tar.gz -C ../tools</run>
+                                </runCmds>
+                                <workdir>/usr/local/skywalking/scenario</workdir>
+                                <cmd>/usr/local/skywalking/run.sh</cmd>
+                            </build>
+                        </image>
+                        <image>
+                            <name>skywalking/agent-test-jvm:jdk14-${BUILD_NO}</name>
```
Comment (wu-sheng) [inline test/plugin/containers/jvm-container/pom.xml:62]: Why I can't find the place to declare using this image? Could you point out?

Comment (dmsolr) [reply test/plugin/containers/jvm-container/pom.xml:62]: It just is used to jdk14 case. And now, I have not enabled it yet.

Comment (dmsolr) [reply test/plugin/containers/jvm-container/pom.xml:62]: I have a big change in building docker image, so I push it for ensuring that is ok.

Comment (wu-sheng) [reply test/plugin/containers/jvm-container/pom.xml:62]: Got it. I was trying to find, how to activate the jdk14 container, notice it is an option only.

File: docs/en/guides/Plugin-test.md [hunk 3]
```diff
@@ -10,26 +10,27 @@ Every plugin maintained in the main repo requires corresponding test cases, also
...
 1. MacOS/Linux
-2. jdk 8+
+2. JDK 8+
 3. Docker
 4. Docker Compose
 
 ## Case Base Image Introduction
 
-The test framework provides `JVM-container` and `Tomcat-container` base images. You could choose the suitable one for your test case, if either is suitable, **recommend choose `JVM-container`**.
+The test framework provides `JVM-container` and `Tomcat-container` base images including JDK8, JDK14. You could choose the suitable one for your test case, if either is suitable, **recommend choose `JVM-container`**.
```
Comment (kezhenxu94) [inline docs/en/guides/Plugin-test.md:19]:
```suggestion
The test framework provides `JVM-container` and `Tomcat-container` base images including JDK8, JDK14. You could choose the suitable one for your test case, if both are suitable, **`JVM-container` is preferred**.
```

**Review by kezhenxu94 — COMMENTED** [review]
Look generally good to me

**Review by kezhenxu94 — APPROVED** [review]
LGTM

**Pull request merged**